docs/new-contributors.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
```

Comment:
Yeah, that's tricky if the links get to long.
For now I shortened the lines, but even if the link exceeded the 80char limit, then I left it as it.